doc/gen/jbuild

```diff
@@ -0,0 +1,15 @@
+(jbuild_version 1)
+
+(alias (
+  (name gen_html)
+  (deps (
+    ../../generator/html/www.exe
+    (files_recursively_in ../static)
+    (files_recursively_in ../templates)
+  ))
+  (action (progn
+    (run ${<})
+    (run rsync -av ../static/ .)
```

Comment:
jbuilder has a builtin rule to copy files http://jbuilder.readthedocs.io/en/latest/jbuild.html#copy-files, could that be used instead of rsync here?

Reply:
Will check.